00:00:00.000294 main     VirtualBox COM Server 6.1.16 r140961 win.amd64 (Oct 16 2020 16:33:17) release log
00:00:00.000297 main     Log opened 2020-11-23T15:55:30.397277300Z
00:00:00.000297 main     Build Type: release
00:00:00.000300 main     OS Product: Windows 10
00:00:00.000301 main     OS Release: 10.0.18363
00:00:00.000302 main     OS Service Pack: 
00:00:00.013890 main     DMI Product Name: 20L6S9U200
00:00:00.020940 main     DMI Product Version: ThinkPad T480
00:00:00.020948 main     Firmware type: BIOS
00:00:00.020952 main     Host RAM: 32426MB (31.6GB) total, 25911MB (25.3GB) available
00:00:00.020956 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.020957 main     Process ID: 14240
00:00:00.020957 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.024368          VirtualBox: object creation starts
00:00:00.024593          Home directory: 'F:\PortableApps\Portable-VirtualBox\data\.VirtualBox'
00:00:00.024684          Installed Drivers:
00:00:00.028448            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.16.40961)
00:00:00.028789            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.16.40961)
00:00:00.029013            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.16.40961)
00:00:00.029418            C:\WINDOWS\System32\drivers\VBoxNetAdp6.sys (Version: 6.1.16.40961)
00:00:00.032214          Loading settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.033774          usbLibGetDevices: Starting USB device enumeration
00:00:00.034979          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.035021          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 7
00:00:00.035056          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 8
00:00:00.035090          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 9
00:00:00.035968          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&d1fe901&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 15
00:00:00.036101          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 1
00:00:00.036147          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 2
00:00:00.036197          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&d2d70c4&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.037573          usbLibGetDevices: Found 10 USB devices, 0 captured
00:00:00.037649          USB: Unknown USB device detected (idVendor: 0x06cb, idProduct: 0x009a)
00:00:00.037664          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x0316)
00:00:00.037680          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x56a6)
00:00:00.037695          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0a2b)
00:00:00.037707          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0ac9)
00:00:00.037715          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0aca)
00:00:00.037725          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0af1)
00:00:00.122045          HostDnsMonitor: initializing
00:00:00.129689          HostDnsMonitor: updating information
00:00:00.129710          HostDnsMonitor: old information
00:00:00.129710            no server entries
00:00:00.129711            no domain set
00:00:00.129711            no search string entries
00:00:00.129712          HostDnsMonitor: new information
00:00:00.129712            server 1: 127.0.0.1
00:00:00.129713            domain: ni.corp.natinst.com
00:00:00.129713            no search string entries
00:00:00.218615          VD: VDInit finished with VINF_SUCCESS
00:00:00.219527          Loading settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:00:00.220330          Loading settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\Debug\Debug.vbox" with version "1.14-windows"
00:00:00.229942          OCI: Local config file 'F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\oci_config' does not exist
00:00:00.230027          OCI: Original config file 'C:\Users\inthirua\.oci\config' does not exist
00:00:00.230028          OCI: Reading profiles finished with status ERROR_SUCCESS
00:00:00.230036          ExtPack: Created cloud provider 'OCI' (hrc=ERROR_SUCCESS)
00:00:00.230057          VirtualBox: object created
00:00:00.688255          Launched VM: 62430352 pid: 4384 (0x1120) frontend: GUI/Qt name: LinuxLive
00:00:04.338059          Load [C:\Program Files\Oracle\VirtualBox\ExtensionPacks\Oracle_VM_VirtualBox_Extension_Pack\win.amd64\VBoxHostWebcam.DLL] vrc=VINF_SUCCESS
00:00:04.575105          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
00:01:22.673604          Saving settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:01:22.838540          Saving settings file "F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\LinuxLive.vbox" with version "1.14-windows"
00:01:28.445402 main     VirtualBox: object deletion starts
00:01:28.861010 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'F:\PortableApps\Portable-VirtualBox\data\.VirtualBox\Machines\LinuxLive\HardDisks/LinuxLive.vmdk' cannot be closed because it is still attached to 2 virtual machines}, preserve=false aResultDetail=0
00:01:28.867558 main     HostDnsMonitor: shutting down ...
00:01:28.873133 main     HostDnsMonitor: shut down
00:01:28.881214 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:01:28.886586 main     VirtualBox: object deleted
